packages/server/.env.example

```diff
@@ -8,8 +8,15 @@ COOKIE_SECRET=itsasecretsecretsecret
 WEBSITE_DOMAIN=http://localhost:8080
 API_DOMAIN=http://localhost:8080
 
+# feature flags 
 ENABLE_GRANTS_SCRAPER=true
 SHARE_TERMINOLOGY_ENABLED=true
+ENABLE_NEW_TEAM_TERMINOLOGY=true
```

Comment:
adding in the other feature flags I've seen used throughout the code for reference in the example env